.github/workflows/cd-compute.yml

```diff
@@ -63,7 +63,7 @@ jobs:
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: ${{ env.AWS_REGION }}
-      - name: Set up crendetials for private ECR
+      - name: Set up credentials for private ECR
```

Comment:
In the future, put the changes impacting CI and CD file configuration into specific commits.